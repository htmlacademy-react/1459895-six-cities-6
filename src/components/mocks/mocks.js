export const сards = [
  {
    type: `Аpartment`,
    price: 120,
    rating: 4.1,
    previewImage: `img/apartment-01.jpg`,
    title: `Beautiful & luxurious studio at great location`,
    isPremium: true,
    isFavorite: true,
    id: 1
  },
  {
    type: `Private room`,
    price: 80,
    rating: 4,
    previewImage: `img/room.jpg`,
    title: `Wood and stone place`,
    isPremium: false,
    isFavorite: false,
    id: 2
  },
  {
    type: `Аpartment`,
    price: 132,
    rating: 4,
    previewImage: `img/apartment-02.jpg`,
    title: `Canal View Prinsengracht`,
    isPremium: false,
    isFavorite: false,
    id: 3
  },
  {
    type: `Аpartment`,
    price: 180,
    rating: 4.8,
    previewImage: `img/apartment-03.jpg`,
    title: `Nice, cozy, warm big bed apartment`,
    isPremium: true,
    isFavorite: true,
    id: 4
  },
  {
    type: `Private room`,
    price: 80,
    rating: 4.2,
    previewImage: `img/room.jpg`,
    title: `Wood and stone place`,
    isPremium: false,
    isFavorite: false,
    id: 5
  }
];
