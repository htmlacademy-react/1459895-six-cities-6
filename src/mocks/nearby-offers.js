const nearbyOffers = [
  {
    type: `Аpartment`,
    price: 120,
    rating: 2.4,
    previewImage: `img/apartment-01.jpg`,
    title: `Beautiful & luxurious studio at great location`,
    isPremium: true,
    isFavorite: true,
    id: 1,
    city: {
      name: `Dusseldorf`,
      location: {latitude: 51.225402, longitude: 6.776314, zoom: 13}
    },
    location: {latitude: 51.233402, longitude: 6.791314, zoom: 16},
  },
  {
    type: `Private room`,
    price: 80,
    rating: 3.8,
    previewImage: `img/room.jpg`,
    title: `Wood and stone place`,
    isPremium: false,
    isFavorite: false,
    id: 2,
    city: {
      name: `Dusseldorf`,
      location: {latitude: 51.225402, longitude: 6.776314, zoom: 13}
    },
    location: {latitude: 51.243402, longitude: 6.791314, zoom: 16},
  },
  {
    type: `Аpartment`,
    price: 160,
    rating: 4,
    previewImage: `img/apartment-02.jpg`,
    title: `Canal View Prinsengracht`,
    isPremium: true,
    isFavorite: true,
    id: 3,
    city: {
      name: `Dusseldorf`,
      location: {latitude: 51.225402, longitude: 6.776314, zoom: 13}
    },
    location: {latitude: 51.228402, longitude: 6.784314, zoom: 16},
  }
];

export {
  nearbyOffers
};
