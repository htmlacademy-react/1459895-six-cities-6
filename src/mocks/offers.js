export const offers = [
  {
    type: `Аpartment`,
    price: 120,
    rating: 2.4,
    previewImage: `img/apartment-01.jpg`,
    title: `Beautiful & luxurious studio at great location`,
    isPremium: true,
    isFavorite: true,
    id: 1,
    bedrooms: 4,
    city: {
      name: `Dusseldorf`,
      location: {latitude: 51.225402, longitude: 6.776314, zoom: 13}
    },
    description: `I rent out a very sunny and bright apartment only 7 minutes walking distance to the metro station. The apartment has a spacious living room with a kitchen, one bedroom and a bathroom with mit bath. A terrace can be used in summer.`,
    goods: [`Fridge`, `Washer`, `Baby seat`, `Laptop friendly workspace`, `Towels`, `Breakfast`, `Air conditioning`],
    host: {
      avatarUrl: `img/avatar-angelina.jpg`,
      id: 25,
      isPro: true,
      name: `Angelina`
    },
    images: [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`
    ],
    location: {latitude: 51.228402, longitude: 6.784314, zoom: 16},
    maxAdults: 7
  },
  {
    type: `Private room`,
    price: 80,
    rating: 4,
    previewImage: `img/room.jpg`,
    title: `Wood and stone place`,
    isPremium: false,
    isFavorite: false,
    id: 2,
    bedrooms: 4,
    city: {
      location: {latitude: 52.37454, longitude: 4.897976, zoom: 13},
      name: `Amsterdam`
    },
    description: `Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.`,
    goods: [`Baby seat`, `Towels`, `Washer`, `Breakfast`, `Laptop friendly workspace`, `Fridge`, `Air conditioning`, `Dishwasher`],
    host: {
      avatarUrl: `img/avatar-angelina.jpg`,
      id: 25,
      isPro: false,
      name: `Angelina`
    },
    images: [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`
    ],
    location: {latitude: 52.361540000000005, longitude: 4.883976, zoom: 16},
    maxAdults: 5
  },
  {
    type: `Аpartment`,
    price: 132,
    rating: 4,
    previewImage: `img/apartment-02.jpg`,
    title: `Canal View Prinsengracht`,
    isPremium: false,
    isFavorite: false,
    id: 3,
    bedrooms: 1,
    city: {
      location: {latitude: 53.550341, longitude: 10.000654, zoom: 13},
      name: `Hamburg`
    },
    description: `I am happy to welcome you to my apartment in the city center! Three words: location, cosy and chic!`,
    goods: [`Laptop friendly workspace`, `Washer`, `Air conditioning`, `Breakfast`],
    host: {
      avatarUrl: `img/avatar-angelina.jpg`,
      id: 25,
      isPro: true,
      name: `Angelina`
    },
    images: [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`
    ],
    location: {latitude: 53.555341000000006, longitude: 9.975654, zoom: 16},
    maxAdults: 2
  },
  {
    type: `Аpartment`,
    price: 180,
    rating: 4.8,
    previewImage: `img/apartment-03.jpg`,
    title: `Nice, cozy, warm big bed apartment`,
    isPremium: true,
    isFavorite: true,
    id: 4,
    bedrooms: 1,
    city: {
      location: {latitude: 51.225402, longitude: 6.776314, zoom: 13},
      name: `Dusseldorf`
    },
    description: `This is a place for dreamers to reset, reflect, and create. Designed with a 'slow' pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.`,
    goods: [`Air conditioning`, `Towels`, `Fridge`, `Breakfast`, `Washer`, `Baby seat`, `Dishwasher`, `Laptop friendly workspace`],
    host: {
      avatarUrl: `img/avatar-angelina.jpg`,
      id: 25,
      isPro: true,
      name: `Angelina`
    },
    images: [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`
    ],
    location: {latitude: 51.243402, longitude: 6.791314, zoom: 16},
    maxAdults: 2
  },
  {
    type: `Private room`,
    price: 80,
    rating: 4.2,
    previewImage: `img/room.jpg`,
    title: `Wood and stone place`,
    isPremium: false,
    isFavorite: true,
    id: 5,
    bedrooms: 1,
    city: {
      location: {latitude: 53.550341, longitude: 10.000654, zoom: 13},
      name: `Hamburg`
    },
    description: `Design interior in most sympathetic area! Complitely renovated, well-equipped, cosy studio in idyllic, over 100 years old wooden house. Calm street, fast connection to center and airport.`,
    goods: [`Baby seat`, `Towels`, `Air conditioning`, `Washer`, `Fridge`, `Coffee machine`, `Laptop friendly workspace`, `Dishwasher`, `Breakfast`],
    host: {
      avatarUrl: `img/avatar-angelina.jpg`,
      id: 25,
      isPro: true,
      name: `Angelina`
    },
    images: [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`
    ],
    location: {latitude: 53.550341, longitude: 9.980654000000001, zoom: 16},
    maxAdults: 1
  }
]
;
