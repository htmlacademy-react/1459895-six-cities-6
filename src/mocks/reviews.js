const reviews = [
  {
    comment: `I stayed here for one night and it was an unpleasant experience.`,
    date: `2021-01-23T08:17:53.220Z`,
    id: 1,
    rating: 3,
    user: {
      avatarUrl: `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/4.jpg`,
      id: 13,
      isPro: false,
      name: `Zak`
    }
  },
  {
    comment: `I stayed here for one night and it was an unpleasant experience.`,
    date: `2021-01-21T08:36:38.657Z`,
    id: 2,
    rating: 3,
    user: {
      avatarUrl: `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/7.jpg`,
      id: 16,
      isPro: true,
      name: `Mollie`
    }
  },
  {
    comment: `Beautiful space, fantastic location and atmosphere, really a wonderful place to spend a few days. Will be back.`,
    date: `2021-01-21T08:36:38.657Z`,
    id: 3,
    rating: 4,
    user: {
      id: 14,
      isPro: true,
      name: `Corey`,
      avatarUrl: `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/5.jpg`
    }
  }
];

export {
  reviews
};
