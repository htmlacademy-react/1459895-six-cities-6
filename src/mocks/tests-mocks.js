export const TEST_DATA = `test`;

export const cards = {
  start: [
    {
      id: 1,
      isFavorite: false
    },
    {
      id: 2,
      isFavorite: false
    },
    {
      id: 3,
      isFavorite: true
    }
  ],
  end: [
    {
      id: 1,
      isFavorite: false
    },
    {
      id: 2,
      isFavorite: true
    },
    {
      id: 3,
      isFavorite: true
    }
  ],
  change: {
    id: 2,
    isFavorite: true
  }
};

export const favoritesCards = {
  start: [
    {
      id: 1,
      isFavorite: true
    },
    {
      id: 2,
      isFavorite: true
    },
    {
      id: 3,
      isFavorite: true
    }
  ],
  end: [
    {
      id: 2,
      isFavorite: true
    },
    {
      id: 3,
      isFavorite: true
    }
  ],
  remove: {
    id: 1,
    isFavorite: true
  }
};

export const offers = {
  adapted: [
    {
      city: {
        name: `Paris`,
        location: {
          latitude: 48.85661,
          longitude: 2.351499,
          zoom: 13
        }
      },
      images: [
        `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
        `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      ],
      goods: [
        `Laptop friendly workspace`,
        `Washer`,
        `Breakfast`,
        `Air conditioning`
      ],
      host: {
        id: 25,
        name: `Angelina`,
        avatarUrl: `img/avatar-angelina.jpg`,
        isPro: true
      },
      location: {
        latitude: 48.85761,
        longitude: 2.358499,
        zoom: 16
      },
      title: `Tile House`,
      rating: 2,
      type: `house`,
      bedrooms: 2,
      price: 161,
      description: `Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.`,
      id: 1,
      previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      isPremium: true,
      isFavorite: false,
      maxAdults: 9
    },
    {
      city: {
        name: `Amsterdam`,
        location: {
          latitude: 48.85661,
          longitude: 2.351499,
          zoom: 13
        }
      },
      images: [
        `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
        `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      ],
      goods: [
        `Laptop friendly workspace`,
        `Washer`,
        `Breakfast`,
        `Air conditioning`
      ],
      host: {
        id: 25,
        name: `Angelina`,
        avatarUrl: `img/avatar-angelina.jpg`,
        isPro: true
      },
      location: {
        latitude: 48.85761,
        longitude: 2.358499,
        zoom: 16
      },
      title: `Tile House`,
      rating: 4,
      type: `house`,
      bedrooms: 3,
      price: 161,
      description: `Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.`,
      id: 2,
      previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      isPremium: true,
      isFavorite: false,
      maxAdults: 3
    }
  ],
  server: [
    {
      "city": {
        "name": `Paris`,
        "location": {
          "latitude": 48.85661,
          "longitude": 2.351499,
          "zoom": 13
        }
      },
      "images": [
        `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
        `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      ],
      "goods": [
        `Laptop friendly workspace`,
        `Washer`,
        `Breakfast`,
        `Air conditioning`
      ],
      "host": {
        "id": 25,
        "name": `Angelina`,
        "avatar_url": `img/avatar-angelina.jpg`,
        "is_pro": true
      },
      "location": {
        "latitude": 48.85761,
        "longitude": 2.358499,
        "zoom": 16
      },
      "title": `Tile House`,
      "rating": 2,
      "type": `house`,
      "bedrooms": 2,
      "price": 161,
      "description": `Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.`,
      "id": 1,
      "preview_image": `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      "is_premium": true,
      "is_favorite": false,
      "max_adults": 9
    },
    {
      "city": {
        "name": `Amsterdam`,
        "location": {
          "latitude": 48.85661,
          "longitude": 2.351499,
          "zoom": 13
        }
      },
      "images": [
        `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
        `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
      ],
      "goods": [
        `Laptop friendly workspace`,
        `Washer`,
        `Breakfast`,
        `Air conditioning`
      ],
      "host": {
        "id": 25,
        "name": `Angelina`,
        "avatar_url": `img/avatar-angelina.jpg`,
        "is_pro": true
      },
      "location": {
        "latitude": 48.85761,
        "longitude": 2.358499,
        "zoom": 16
      },
      "title": `Tile House`,
      "rating": 4,
      "type": `house`,
      "bedrooms": 3,
      "price": 161,
      "description": `Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.`,
      "id": 2,
      "preview_image": `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
      "is_premium": true,
      "is_favorite": false,
      "max_adults": 3
    }
  ]
};

export const reviews = {
  server: [
    {
      "user": {
        "id": 11,
        "name": `Jack`,
        "avatar_url": `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/2.jpg`,
        "is_pro": false
      },
      "id": 1,
      "rating": 3,
      "comment": `I stayed here for one night and it was an unpleasant experience.`,
      "date": `2021-02-18T15:34:01.286Z`
    },
    {
      "user": {
        "id": 11,
        "name": `Jack`,
        "avatar_url": `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/2.jpg`,
        "is_pro": false
      },
      "id": 2,
      "rating": 4,
      "comment": `The deluxe room was a quite comfortable one with all the adequate facilities. The only thing that made me feel uncomfortable was the rude behavior of an impolite staff at the reception desk.`,
      "date": `2021-02-18T15:34:01.286Z`
    }
  ],
  adapted: [
    {
      user: {
        id: 11,
        name: `Jack`,
        avatarUrl: `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/2.jpg`,
        isPro: false
      },
      id: 1,
      rating: 3,
      comment: `I stayed here for one night and it was an unpleasant experience.`,
      date: `2021-02-18T15:34:01.286Z`
    },
    {
      user: {
        id: 11,
        name: `Jack`,
        avatarUrl: `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/2.jpg`,
        isPro: false
      },
      id: 2,
      rating: 4,
      comment: `The deluxe room was a quite comfortable one with all the adequate facilities. The only thing that made me feel uncomfortable was the rude behavior of an impolite staff at the reception desk.`,
      date: `2021-02-18T15:34:01.286Z`
    }
  ]
};

export const offer = {
  adapted: {
    city: {
      name: `Paris`,
      location: {
        latitude: 48.85661,
        longitude: 2.351499,
        zoom: 13
      }
    },
    images: [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`
    ],
    goods: [
      `Laptop friendly workspace`,
      `Washer`,
      `Breakfast`,
      `Air conditioning`
    ],
    host: {
      id: 25,
      name: `Angelina`,
      avatarUrl: `img/avatar-angelina.jpg`,
      isPro: true
    },
    location: {
      latitude: 48.85761,
      longitude: 2.358499,
      zoom: 16
    },
    title: `Tile House`,
    rating: 2,
    type: `house`,
    bedrooms: 2,
    price: 161,
    description: `Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.`,
    id: 1,
    previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
    isPremium: true,
    isFavorite: false,
    maxAdults: 9
  },
  server: {
    "city": {
      "name": `Paris`,
      "location": {
        "latitude": 48.85661,
        "longitude": 2.351499,
        "zoom": 13
      }
    },
    "images": [
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
      `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
    ],
    "goods": [
      `Laptop friendly workspace`,
      `Washer`,
      `Breakfast`,
      `Air conditioning`
    ],
    "host": {
      "id": 25,
      "name": `Angelina`,
      "avatar_url": `img/avatar-angelina.jpg`,
      "is_pro": true
    },
    "location": {
      "latitude": 48.85761,
      "longitude": 2.358499,
      "zoom": 16
    },
    "title": `Tile House`,
    "rating": 2,
    "type": `house`,
    "bedrooms": 2,
    "price": 161,
    "description": `Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.`,
    "id": 1,
    "preview_image": `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
    "is_premium": true,
    "is_favorite": false,
    "max_adults": 9
  },
};

export const authInfo = {
  server: {
    "id": 1,
    "email": `mail@gmail.com`,
    "name": `alex`,
    "avatar_url": `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/3.jpg`,
    "is_pro": false
  },
  adapted: {
    id: 1,
    email: `mail@gmail.com`,
    name: `alex`,
    avatarUrl: `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/3.jpg`,
    isPro: false
  }
};

export const options = [`Popular`, `Price: low to high`, `Price: high to low`, `Top rated first`];
