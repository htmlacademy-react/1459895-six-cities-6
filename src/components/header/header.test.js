import React from 'react';
import {render} from '@testing-library/react';
import Header from './header';
import {authInfo} from "../../mocks/tests-mocks";
import * as redux from "react-redux";
import configureStore from "redux-mock-store";
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';

const mockStore = configureStore({});
describe(`Test Header component`, () => {
  it(`Header component should render correctly`, () => {
    const store = mockStore({
      USER: {authInfo: null}
    });
    const history = createMemoryHistory();
    const {container} = render(
        <redux.Provider store={store}>
          <Router history={history}>
            <Header />
          </Router>
        </redux.Provider>
    );
    expect(container).toMatchSnapshot();
  });
  it(`Header component should render correctly when user authorized`, () => {
    const store = mockStore({
      USER: {authInfo: authInfo.adapted}
    });
    const history = createMemoryHistory();
    const {container} = render(
        <redux.Provider store={store}>
          <Router history={history}>
            <Header />
          </Router>
        </redux.Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
