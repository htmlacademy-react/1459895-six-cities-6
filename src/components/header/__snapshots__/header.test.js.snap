// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header component Header component should render correctly 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="container"
    >
      <div
        class="header__wrapper"
      >
        <div
          class="header__left"
        >
          <a
            class="header__logo-link header__logo-link--active"
            href="/"
          >
            <img
              alt="6 cities logo"
              class="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          class="header__nav"
        >
          <ul
            class="header__nav-list"
          >
            <li
              class="header__nav-item user"
            >
              <a
                class="header__nav-link header__nav-link--profile"
                href="/login"
              >
                <span
                  class="header__user-name user__name"
                >
                  Sign In
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
</div>
`;

exports[`Test Header component Header component should render correctly when user authorized 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="container"
    >
      <div
        class="header__wrapper"
      >
        <div
          class="header__left"
        >
          <a
            class="header__logo-link header__logo-link--active"
            href="/"
          >
            <img
              alt="6 cities logo"
              class="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          class="header__nav"
        >
          <ul
            class="header__nav-list"
          >
            <li
              class="header__nav-item user"
            >
              <a
                class="header__nav-link header__nav-link--profile"
                href="/favorites"
              >
                <div
                  class="header__avatar-wrapper user__avatar-wrapper"
                />
                <span
                  class="header__user-name user__name"
                >
                  mail@gmail.com
                </span>
              </a>
            </li>
            <li
              class="header__nav-item user"
            >
              <button
                class="header__nav-link header__nav-link--profile header__nav-link--logout"
              >
                <span
                  class="header__user-name user__name"
                >
                  Logout
                </span>
              </button>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
</div>
`;
